### ISSUE TITLE
Pretrain on data from Weather4cast

### ISSUE THREAD

**Issue by jacobbieker** [openclimatefix/satflow#100]
## Detailed Description
https://www.iarai.ac.at/weather4cast/ (on GitHub here: https://github.com/iarai/weather4cast) has some nice weather data from around the world, including cloud masks, all at 4km resolution, so similar to what EUMETSAT gives. This could be useful for pretraining any of the models before finetuning more on our specific data.

## Context
Pretraining has been proven to help quite a bit in large models, so this might help there.

Interestingly, they use MSE as their loss function for it all, even though its a video prediction task, and they want the next 8 hours of data predicted. So maybe MSE isn't the worst?

## Possible Implementation
Download the data, and run some models on it.

**Comment by jacobbieker** [comment]
Actually, this is EUMETSAT data! Including some of the optimum cloud masks, and I am assuming the 15 minute full disk images. So for transfer learning this would actually probably be quite useful?

**Comment by codeastra2** [comment]
If no one is working on this, I would like to take this as a  first issue!

**Comment by jacobbieker** [comment]
Yeah, that sounds great!

**Comment by codeastra2** [comment]
So to confirm https://github.com/iarai/weather4cast/blob/master/utils/1.%20Onboarding.ipynb these are the ses that need to be done and the h5 files need to be uploaded?

**Comment by jacobbieker** [comment]
Yeah, that would be great! Ideally, they could be uploaded to HuggingFace Datasets so they are easily widely available for anyone to use.
